### DISCUSSION TITLE
Log HTTP errors in InfluxDB instead of Sentry

### DISCUSSION THREAD

**Discussion by raphael0202** [openfoodfacts/robotoff#1232] [Ideas]
Currently, all HTTP errors (4XX, 5XX) - mostly when calling Open Food Facts (Product Opener) API - are logged in Sentry.
This type of error occurs frequently, and Sentry is not the best tool to keep track of this kind of error.
Instead of logging it in Sentry, log it in in Influx DB (probably in monitoring infuxDB).

**Comment by AnshMasand** [comment]
hello, do I need to setup an influxdb cloud account to solve this issue? or am I looking for answers in the wrong area?

**Comment by raphael0202** [comment]
Hello!
No we already have 2 InfluxDB database instances running, one for monitoring and one for business metrics. Currently Robotoff is only connected to the "metrics" InfluxDB (see metrics.py).

**Comment by alexgarel** [comment]
Connection to the monitoring influxdb is possible. In staging and production it's exposed on 10.0.0.3:8086 see ([docker-compose](https://github.com/openfoodfacts/openfoodfacts-monitoring/blob/main/docker-compose.yml#L15) and github action), but of course, this is a private network.

If you want to test locally you should use docker to launch an influxdb locally and expose it.
